### DISCUSSION TITLE
False virus alert?

### DISCUSSION THREAD

**Discussion by smk422** [PlummersSoftwareLLC/HelloAssembly#32] [General]
Hi all, after building Tiny.asm, Microsoft Defender detects virus:

![tiny_asm_virus](https://github.com/PlummersSoftwareLLC/HelloAssembly/assets/89627536/6c4c9484-4814-4f49-8fd5-101f5195b255)

I guess this is a false alarm. My Windows 11 machine seems to be ok after a full Defender scan.  Anybody else with same experience?

**Comment by davepl** [comment]
It does some funky stuff that I guess is also common to malware, but tiny is just trying to be small, not dangerous.  Should be fine to ignore or whitelist.

**Comment by rbergen** [comment]
What Dave said. It triggers certain virus scanners - and Defender seems to be particularly sensitive - but it’s harmless.

**Reply by smk422** [reply]
Thanks for replies, I got the answer I was hoping for :-)
